data/building-hours/1-2-pause-kitchen.yaml

```diff
@@ -6,9 +6,10 @@ schedule:
   - title: Daylight
     notes: The late night menu starts at 9 p.m.
     hours:
-      - {days: [Mo, Tu, We, Th], from: '10:30am', to: '12:00am'}
-      - {days: [Fr, Sa], from: '10:30am', to: '9:00pm'}
-      - {days: [Su], from: '10:30am', to: '12:00am'}
+      - {days: [Mo, We], from: '10:30am', to: '12:00am'}
+      - {days: [Fr], from: '10:30am', to: '9:00pm'}
+      - {days: [Sa], from: '11:30am', to: '9:00pm'}
+      - {days: [Su, Tu, Th], from: '11:30am', to: '12:00am'}
```

Comment:
Do we want Sunday, Tuesday, Thursday,

Or Tuesday, Thursday, Sunday?

Alternately, I would propose splitting Tu/Th and Su into different schedules, even though they're identical, for sorting in the list:

Mo, We
Tu, Th
Fr
Sa
Su

...

Going off of that, what about just breaking each day into its own schedule, and going Mo, Tu, We, Th, Fr, Sa, Su separately?

The user is already going to have to scan a lot to figure out what day it is on the schedule.